### ISSUE TITLE
Spice Handlebars Helpers: Make sure helper names are pre-fixed with Spice name

### ISSUE THREAD

**Issue by moollaza** [duckduckgo/zeroclickinfo-spice#1480]
Right now all our handlebars helpers are defined in a global space, which mean if two IA's trigger and define a helper with the same name, we're going to run into issues.

We will soon be fixing this on a lower, platform level, but for now we need to ensure any helpers that are defined have unique names. The easiest way to ensure this is to prefix the name of the Spice onto the function's name:

e.g.
```js
Spice.registerHelper("ia_name_function_name", function() {
    //do stuff
});
```

If any helpers can easily be replaced, with local function that are executed in the IA's `normalize()` function, that's even better as we don't have to bother with the extra code in the template.

------

Here are some helpers that need to be addressed:

- [x] airlines/airlines.js: `Handlebars.registerHelper("relative",` - #1498
- [x] betterific/betterific.js:H`andlebars.registerHelper('dashedS',`
- [x] detect_lang/detect_lang.js: `Handlebars.registerHelper("expandLang",`
- [x] go_watch_it/go_watch_it.js: `Spice.registerHelper("buyOrRent",`
- [x] movie/movie.js: `Handlebars.registerHelper("time", function(runtime)`
- [x] open_snp/open_snp.js:H`andlebars.registerHelper('facts',`
- [x] translate/from_to/translate_from_to.js: `Handlebars.registerHelper('getTerms',`

**Comment by MrChrisW** [comment]
@moollaza I've taken a quick look at the Airlines IA. 
Please correct me if I'm wrong but I don't believe the following [handlebars helpers](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/airlines/airlines.js#L165-L203) (`relative`, `airline_status)` are actually used by the IA, see [content.handlebars](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/airlines/content.handlebars).

**Comment by moollaza** [comment]
@MrChrisW looks like you're right! Good catch. Feel free to remove that stuff. I'm always happy to remove old/unused code :+1:

**Comment by MrChrisW** [comment]
Fixed in #1500
